src/components/select/Select.tsx

```diff
@@ -43,14 +44,16 @@ export type SelectProps<
   onChange: (changed: Option | null, actionMeta: ActionMeta<Option>) => void;
 } & Omit<Props<Option, IsMulti, Group>, 'isMulti' | RequiredProps> &
   Required<Pick<Props<Option, IsMulti, Group>, RequiredProps>> &
-  HelpTextProps;
+  HelpTextProps &
+  BaseProps;
 
 export const Select = <
   Option extends SelectOption,
   Group extends GroupBase<Option> = GroupBase<Option>,
 >({
   isClearable = true,
   label,
+  style,
```

Comment:
```suggestion
```
Maybe this component doesn't need style since it doesn't have any children inside